engines/bladerunner/bladerunner.cpp

```diff
@@ -323,6 +329,20 @@ bool BladeRunnerEngine::startup(bool hasSavegames) {
 	_gameFlags = new GameFlags();
 	_gameFlags->setFlagCount(_gameInfo->getFlagCount());
 
+#if BLADERUNNER_RESTORED_CONTENT_GAME
+	// EDS flags
+	_extraGameFlagsForRestoredContent = new GameFlags(); //aux flags - custom
+	_extraGameFlagsForRestoredContent->setFlagCount(RESTORED_CONTENT_EXTRA_FLAGS + 1); // aux flags - custom // +1 since we don't assign something to 0 enum value
+#if SUBTITLES_SUPPORT
+#if SUBTITLES_ENABLED_BY_DEFAULT
```

Comment:
Instead of making it compile-time, you could just read from our common GUI option, where we have subtitles settings. Just start using it for the engine.

Reply:
No, please don't remove them. Rather synchronise both (see syncSoundSettings() method).